pychunkedgraph/backend/chunkedgraph.py

```diff
@@ -2432,6 +2433,11 @@ def get_roots(self, node_ids: Sequence[np.uint64],
                 return parent_ids
             else:
                 time.sleep(0.5)
+        # if we have failed to return by now, then we we have some roots
+        # that have failed to get to the stop layer as if we want to assert_roots
+        # then we should fail that assertion
+        if assert_roots:
+            assert False
```

Comment:
This could just be `assert not assert_roots` or even better an exception :)

Reply:
Exception with message is better, `assert not assert_roots` can be confusing

Reply:
Fixed